server/index.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require('supertest');
+const app = require('./index');
+
+describe('test root path', () => {
+  test('it should respond with status code 200', () => {
+    return request(app).get('/').expect(200);
+  });
+});
+
+describe('test featured item get request', () => {
```

Comment:
This appears to be an integration tests amongst unit tests - you may want to mock the data being retrieved from your database.